### PR TITLE
[Form] Add missing Latvian translations

### PR DESCRIPTION
**Pull request by andrewmy**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| Tickets       | Part of #34710, #34749
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf [hunk 1]
```diff
@@ -14,6 +14,126 @@
...
+                <source>The hidden field is invalid.</source>
+                <target>Slēptā lauka vērtība ir nekorekta.</target>
+            </trans-unit>
+            <trans-unit id="111">
+                <source>Please enter an integer.</source>
+                <target>Lūdzu, ievadiet veselu skaitli.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Lūdzu, izvēlieties korektu valodu.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: Is "korektu" the right word here?

How about "derīgu"?

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf [hunk 2]
```diff
@@ -14,6 +14,126 @@
...
+                <source>Please enter a valid time.</source>
+                <target>Lūdzu, ievadiet korektu laiku.</target>
+            </trans-unit>
+            <trans-unit id="120">
+                <source>Please select a valid timezone.</source>
+                <target>Lūdzu, izvēlieties korektu laika zonu.</target>
+            </trans-unit>
+            <trans-unit id="121">
+                <source>Please enter a valid URL.</source>
+                <target>Lūdzu, ievadiet korektu tīmekļa adresi.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: It is okey to use the technical term URL here. 

A "link" or a "web address" is not exactly the same.

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf [hunk 3]
```diff
@@ -14,6 +14,126 @@
...
+            </trans-unit>
+            <trans-unit id="122">
+                <source>Please enter a valid search term.</source>
+                <target>Lūdzu, ievadiet korektu meklēšanas atslēgvārdu.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: Make sure this is a "search term". ie, a sentence or a few words. 

If this translates to "keyword", doesnt that mean just one word.

**Review by Nyholm — COMMENTED** [review]
Thank you very much. 

I've used google translate to confirm this. My questions below might be way wrong. But I want you to just double check these 3.

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf (hunk 1 above)
Comment (andrewmy) [reply src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: Both would be acceptable but you're right, derīgu sounds better :)

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf (hunk 3 above)
Comment (andrewmy) [reply src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: Your Latvian is suspiciously good :)

Comment (Nyholm) [reply src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: hehe. Thank you =)

File: src/Symfony/Component/Form/Resources/translations/validators.lv.xlf (hunk 2 above)
Comment (andrewmy) [reply src/Symfony/Component/Form/Resources/translations/validators.lv.xlf]: OK

**Review by Nyholm — APPROVED** [review]
Thank you

Im happy with this PR :+1:

**Comment by derrabus** [comment]
Thank you @andrewmy.

**Pull request merged**